### PR TITLE
serverless support cluster events

### PR DESCRIPTION
**Pull request by zuowei593**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.2 (TiDB 7.2 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by srstack — APPROVED** [review]

**Review by baiyuqing — APPROVED** [review]
LGTM

**Review by qqqdan — APPROVED** [review]

File: tidb-cloud/tidb-cloud-events.md [hunk 1]
```diff
@@ -55,10 +53,6 @@ For each event, the following information is logged:
...
 - Triggered By
 
-> **Note:**
->
-> Cluster events started before 2023-03-22 are not visible on the Events page.
-
 ## Event retention policy
 
-For TiDB Dedicated clusters, the event data is kept for 7 days.
+Event data are kept for 7 days.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-events.md:58]:
```suggestion
Event data is kept for 7 days.
```

Comment (qiancai) [reply tidb-cloud/tidb-cloud-events.md:58]: Google developer documentation style guide https://developers.google.cn/style/word-list#data
image

**Review by qiancai — APPROVED** [review]

File: tidb-cloud/tidb-cloud-events.md [hunk 2]
```diff
@@ -46,6 +42,8 @@ TiDB Cloud logs the following types of cluster events:
 | ScaleChangefeed |   Scale the specification of a changefeed |  
 | FailedChangefeed |   Changefeed failures |  
 | ImportData |   Import data to a cluster |  
+| UpdateSpendingLimit |   Update serverless cluster spending limit |  
+| ResourceLimitation |   Update serverless cluster resource limitation |
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-events.md:46]:
```suggestion
| UpdateSpendingLimit |   Update spending limit of TiDB Serverless cluster |  
| ResourceLimitation |   Update resource limitation of TiDB Serverless cluster |  
```

File: tidb-cloud/tidb-cloud-events.md [hunk 3]
```diff
@@ -46,6 +42,8 @@ TiDB Cloud logs the following types of cluster events:
 | ScaleChangefeed |   Scale the specification of a changefeed |  
 | FailedChangefeed |   Changefeed failures |  
 | ImportData |   Import data to a cluster |  
+| UpdateSpendingLimit |   Update spending limit of a TiDB Serverless cluster |  
+| ResourceLimitation |   Update resource limitation of a TiDB Serverless cluster |
```
Comment (Oreoxmt) [inline tidb-cloud/tidb-cloud-events.md:46]:
```suggestion
| ResourceLimitation |   Update resource limit of a TiDB Serverless cluster |  
```

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**